optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/api/PlannerBenchmarkFactory.java

```diff
@@ -131,7 +131,7 @@ public static PlannerBenchmarkFactory createFromKieContainerXmlResource(ReleaseI
      * @return never null
      */
     public static PlannerBenchmarkFactory createFromKieContainerXmlResource(KieContainer kieContainer,
-                                                                            String benchmarkConfigResource) {
+            String benchmarkConfigResource) {
```

Comment:
Is this the proper formatting of multi-line arguments? If so, I have to reconfigure my IDE.

Reply:
It's an open, painful discussion. At some point it was 2 indents in the kie standard - and it was in sync with the old Java Sun standards and Jboss standards - but then someone changed it to align on column - going out of sync with most of the broader Java community.

Honestly, align on columns sucks big time, especially for files with advanced generics, so in the optaplanner repo, we're doing 2 indents... Let's talk about it some time.